ui/service.go

```diff
@@ -531,10 +531,12 @@ func (us *uiService) Index(token string) (b []byte, err error) {
 		NavbarActive   string
 		CollapseActive string
 		Summary        dataSummary
+		SessionDetails Session
```

Comment:
Let's keep this aligned
```suggestion
		Session Session
```